2021/OffSeason2021/src/main/include/commands/ShootingAction.h

```diff
@@ -9,6 +9,12 @@
 // it from being updated in the future.
 
 #pragma once
+/*
+#include "subsystems/FloorConveyor.h"
+#include "subsystems/Intake.h"
+#include "subsystems/Shooter.h"

```

Comment:
Commenting these out is fine for debugging purposes, but when you make a PR go ahead and just delete things like this.  We can always add them back later